src/styles/blocks/catalog.scss

```diff
@@ -10,16 +10,22 @@
     display: flex;
     justify-content: center;
     align-items: center;
-    box-sizing: border-box;
+    width: $card-width;
+    height: $card-height;
+    margin-bottom: $card-margin-bottom;
 
-    width: 200px;
-    height: 300px;
     border: 1px solid #ccc;
     border-radius: 5px;
-    margin-bottom: 48px;
+
+    transition: transform 0.3s ease;
```

Comment:
You should mention all changed properties

Reply:
done